democracy_club/templates/about.html

```diff
@@ -81,16 +80,17 @@
 
 ### Who are we?
 
-Currently, the club is coordinated by [Sym Roe](https://twitter.com/symroe) and
-[Joe Mitchell](https://twitter.com/j0e_m) as part of the
-Democracy Club Group. There is also a strongly motivated core volunteer group
-of 10-20 individuals who assist.
+Currently, the club is coordinated by Sym and Joe.
+
+![Joe and Sym, Democracy Club coordinators](https://s14.postimg.org/n8t6twmk1/Joe_Sym_with_names_800x500.jpg "Sym and Joe")
```

Comment:
Apologies! I fail at markdown syntax. I didn’t realise where the title text went. V nice pic!